impl/sql/mutationstorage/queue.go

```diff
@@ -101,57 +102,70 @@ func (r *Receiver) run(ctx context.Context, last time.Time) {
 	}
 
 	if time.Since(last) > (r.opts.MaxPeriod - r.opts.Period) {
-		r.sendBatch(ctx, 0, r.opts.MaxBatchSize) // We will be overdue for an epoch soon.
+		if _, err := r.sendMultiBatch(ctx, 0, int(r.opts.MaxBatchSize)); err != nil {
+			glog.Errorf("firstTick: sendMultiBatch(): %v", err)
```

Comment:
This error doesn't get handled. Happens several times below. I assume it's intentional but looks suspicious.

Reply:
It is indeed intentional. Run is at the top of it's call stack, so there's not really any other place to go from here.